include/ignition/math/detail/Box.hh

```diff
@@ -113,6 +121,165 @@ T Box<T>::Volume() const
   return this->size.X() * this->size.Y() * this->size.Z();
 }
 
+//////////////////////////////////////////////////
+/// \brief Given a *convex* polygon described by the verices in a given plane,
+/// compute the list of triangles which form this polygon.
+/// \param[in] _plane The plane in which the vertices exist.
+/// \param[in] _vertices The vertices of the polygon.
+/// \return a list of triangles, or and empty vector if _vertices in the _plane
```

Comment:
spelling: "or *an* empty vector"

also, can you document the scalar 2nd value in the `std::pair`?

Reply:
b4f9ddc